packages/semcom-node/lib/main.ts

```diff
@@ -1,6 +1,6 @@
 import * as path from 'path';
 import { ComponentsManager } from 'componentsjs';
-import { LauncherService } from './launcher/services/launcher.service';
+import { NodeHttpServer } from '@digita-ai/handlersjs-http';
 
 const start = async () => {
```

Comment:
In the proxy package we called this `launch`, to avoid confusion with the server start call.
```suggestion
const launch = async () => {
```